backend/internal/server/apis/league/league.go

```diff
@@ -25,3 +27,23 @@ func getLeaguesHandler(c *fiber.Ctx) error {
 
 	return responder.OkWithData(c, leagues)
 }
+
+func postLeaguesHandler(c *fiber.Ctx) error {
+	log := locals.Logger(c)
+
+	leaguesRequest := &models.League{}
+	err := c.BodyParser(leaguesRequest)
+	if err != nil {
+		log.WithErr(err).Alert("Failed to parse leagues request payload")
+		return responder.BadRequest(c, "Failed to parse leagues request payload")
+	}
+
+	db := db.GetSession(c)
+	err = db.CreateLeagues(leaguesRequest)
```

Comment:
💡 One league or multiple?

Reply:
I agree, I would change this to singular.